Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.cpp

```diff
@@ -1098,6 +1229,18 @@ namespace AzToolsFramework
             });
     }
 
+    template<typename Vertex>
+    EditorVertexSelectionVariable<Vertex>::EditorVertexSelectionVariable()
+    {
+        EditorVertexSelectionRequestBus::Handler::BusConnect();
+    }
+
+    template<typename Vertex>
+    EditorVertexSelectionVariable<Vertex>::~EditorVertexSelectionVariable()
+    {
+        EditorVertexSelectionRequestBus::Handler::BusDisconnect();
```

Comment:
question: This must have been being connected before? 🤔 Were these moved?

Reply:
I created the bus in this PR so it did not exist before.